README.md

```diff
@@ -1,4 +1,6 @@
+[![Build Status](https://travis-ci.org/Mifrill/railgun.svg?branch=master)](https://travis-ci.org/Mifrill/railgun)
```

Comment:
Status for this repo shouldn't display status from your repo. It should display coverage for zharikovpro/railgun repo, or skip it.